app/views/security_check_failed/show.html.erb

```diff
@@ -0,0 +1,27 @@
+<% self.title = t('security_check_failed.title') %>
+
+<%= render AlertIconComponent.new(icon_name: :warning, class: 'display-block margin-bottom-4') %>
+<%= render PageHeadingComponent.new.with_content(t('security_check_failed.title')) %>
+<p>
+  <%= t('security_check_failed.details_html') %>
+  <%= new_tab_link_to(
+        t('security_check_failed.learn_more', app_name: APP_NAME),
+        help_center_redirect_path(
+          category: 'trouble-signing-in',
+          article: 'security-check-failed',
+        ),
+      ) %>
+</p>
+
+<ul class="usa-list margin-top-0">
+  <li><%= t('security_check_failed.info_p1', app_name: APP_NAME) %></li>
+  <li><%= link_to(
+            t('security_check_failed.info_p2_link'),
+            login_piv_cac_url,
+          ) + ' ' + t('security_check_failed.info_p2_part2') %></li> 
```

Comment:
I think it might make sense to embed the link within a single localized string rather than combine two strings, since some languages may need to rearrange the words such that the link will not always necessarily be at the beginning of the sentence.